### DISCUSSION TITLE
Looking for recomendations for a haswell chromebook recovery match better then coral?

### DISCUSSION THREAD

**Discussion by madhits** [sebanc/brunch#2155] [Q&A]
So I have been on rammus up until the kernel 6 jump. Once 6x kernel I can no longer boot rammus, or maybe it was other changes, but it was something that drastically changed with r117. Last version was r116 on k 5.15. I have a system that matches specs to coral almost exact match to babytiger if that helps. I can boot any coral image but coral for some reason never got arcvm (android 11), and was stuck on arc++(android 9) up until its support ended with r126. So I'm wondering if someone out there with haswell based machine running a recovery that boots above 117 with arcvm working. If your on a 4th gen intel machine I'd love to know what recovery's you were able to boot. One person got octopus 117 but its not much of a jump to bother with. I'm hoping someone got something like a 128 going with arcvm on haswell. Maybe @sebanc can give some details as to why arcvm works with kernal 5.15 but most of the time only works with kernel 6x only. Also love to hear from @signigelchan & @tejasraman if you have any input.

**Comment by tejasraman** [comment]
IIRC ArcVM (officially) works on kernels as old as 4.14 (if not older) but I'm not really sure. ArcVM was rolled out in phases so maybe try a more modern image (octopus DEFINITELY has ArcVM) but I don't really have better suggestions. 

Sadly I don't have a Haswell machine to test with..

**Reply by madhits** [reply]
Thanks for the input, ok so arcvm is not kernel specific. I think 3rd & 4th gen core processors seem to work with the same recoveries a higher % of the time. I tried octopus 128 and it did not boot, I doubt there is much I can do if it wont get around the chrome OS screen freeze is there? Thanks for the info and if you have any tips to get octopus to boot im all for trying it.
